ginga/util/iohelper.py

```diff
@@ -18,8 +18,10 @@ def get_fileinfo(filespec, cache_dir='/tmp', download=False):
     """
     Parse a file specification and return information about it.
     """
-    idx = None
-    name_ext = ''
+    # Loads first science extension by default.
+    # This prevents [None] to be loaded instead.
+    idx = ('SCI', 1)
+    name_ext = '[SCI,1]'
```

Comment:
To clarify, I think I can modify `io_fits` to return the `numhdu` as (in your case) "[SCI, 1]" if `numhdu` is passed in as `None`.